pkgs/tools/X11/smart-wallpaper/default.nix

```diff
@@ -5,17 +5,18 @@
 , xdpyinfo
 , killall
 , xwinwrap
+, swaybg
```

Comment:
Shall we add this to the wrapper as well?

Reply:
@bobby285271 Oops, just added that. Thank you!